docs/index.md

```diff
@@ -1307,6 +1311,9 @@ The `Application` object is instantiated with the following settings:
 - `types`: A list of all resource types declared and handled by this app.
 - `processors`: If you define custom processors, they have to be registered here as instances.
 - `defaultProcessor`: All non-bound-to-processor resources will be handled by this processor.
+- `transportLayerOptions`: Allows to customize some aspects of the transport layer used by the application.
+  - `httpBodyPayload`: Describes how big the request's `body` can be, expressed in a string i.e. `10mb`. Defaults to `1mb`.
```

Comment:
Isn't this a middleware concern?
For example, in Next.js it's defined inside api route file, a.k.a `api/[...kurier].js`
https://nextjs.org/docs/api-routes/api-middlewares#custom-config

Reply:
It's been there since `1.0.0`. We wanted to keep the jsonApi* middlewares as simple as possible and `transportLayerOptions` seemed like a nice place back then.

Revisiting the idea, we might want to add support for also configuring `transportLayerOptions` directly into the middleware. To prevent breaking changes for now, we'd support both ways.

Reply:
something to add to v2 maybe? We can break compatibility then 😉

Reply:
Sounds good